mlir/include/mlir/Dialect/LLVMIR/ROCDLOps.td

```diff
@@ -158,6 +158,16 @@ Arguments<(ins I32:$index,
    }];
 }
 
+def ROCDL_BallotOp :
+  ROCDL_Op<"ballot">,
+  Results<(outs LLVM_Type:$res)>,
```

Comment:
This is missing a documentation: please add a summary and complete description of the op semantics. We'd want the MLIR website to be completely documented.

Reply:
Good point. I see now that none of the other intrinsics op have descriptions in the file, but it is not a reason not to add it. @zahimoud can you send a follow up PR?